test.py

```diff
@@ -0,0 +1,47 @@
+import os
+
```

Comment:
I'd avoid calling this file `test.py` as readers will assume it's a test in the formal sense.

My suggestion would be to create a folder called `scripts` and put this file inside it named something like `make_example_request.py`

Reply:
Buuut see comment below - this file should probably be converted to actual unit tests

Reply:
Moved the current test.py to scritps/make_example_requests.py and added instructions.